charts/xteve/values.yaml

```diff
@@ -0,0 +1,121 @@
+# Default values for xteve.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# upgrade strategy type (e.g. Recreate or RollingUpdate)
+strategyType: Recreate
+
+image:
+  repository: tnwhitwell/xteve
+  tag: 2.1.2.0121-alpine3.12.0-curl
+  pullPolicy: IfNotPresent
+
+# timezone to run the service as
+# timezone: "America/New York"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# Probes configuration
+probes:
+  liveness:
+    initialDelaySeconds: 30
+    failureThreshold: 5
+    timeoutSeconds: 10
+  readiness:
+    initialDelaySeconds: 30
+    failureThreshold: 5
+    timeoutSeconds: 10
+
+hostNetwork: false
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+persistence:
+  config:
+    enabled: true
```

Comment:
It's minor, but I wonder if we should default the persistence to false for a default value?  Nearly every other chart I've come across that has a persistence option usually has the default setting to false.

Reply:
Good catch, I'll make the changes in the morning.